exercises/practice/change/.meta/proof.ci.wren

```diff
@@ -17,34 +20,33 @@ class Change {
    *
    * S = the _first_ coin used to make change for amount n.
    */
-  static change(d, n) {
-    var C = [0] + List.filled(n, Num.largest)
-    var S = [0] + List.filled(n, null)
+  static change(denominations, valueOfCoins) {
+    var C = [0] + List.filled(valueOfCoins, null)
```

Comment:
I was using the same variables as the paper, but yes this is more readable.

Reply:
Yeah, I like the proofs to be legible. :-)  I think it's still pretty clear it follows the paper at a glance.